README.md

```diff
@@ -2,9 +2,9 @@
 
 This `design-notes` repository contains design proposals, enhancements, and architectural decisions for Radius. It provides a consistent and controlled path to record changes and developments, ensuring clarity and transparency for all stakeholders in the Radius community.
 
-Design notes are used for evaluating the design of **planned** changes and additions to Radius. Do not use this repository for feature requests - please create an issue on the [main Radius repository](https://github.com/radius-project) instead. We create documents in this repo to describe **how** to accomplish something once we have agreement that it is appropriate for the project.
+Design notes are used for evaluating the design of **planned** changes and additions to Radius. Do not use this repository for feature requests - please create an issue on the [main Radius repository](https://github.com/radius-project) instead. We create documents in this repo to describe **why** and **how** to accomplish something once we have agreement that it is appropriate for the project.
 
-Minor changes such as documentation updates or small bug fixes may be reviewed and implemented directly via a GitHub issue. For larger changes, such as new feature design, a design note pull-request and review is required. These must get consensus from Radius maintainers and the community, and should follow the [design note template](./template/YYYY-MM-design-template.md) in this repository.
+Minor changes such as documentation updates or small bug fixes may be reviewed and implemented directly via a GitHub issue. For larger changes, such as new feature design, a design note pull-request and review is required. These must get consensus from Radius maintainers and the community. A product specification following the [product spec template](./template/YYYY-MM-product-spec.md) should precede the [design note template](./template/YYYY-MM-design-template.md).
```

Comment:
Very minor nitpick from me. Could we call this a `feature spec`?

Radius is a **project** not a **product**.

Reply:
Ooh yeah feature spec sounds good to me. @willtsai/ @jonvsm  - thoughts?

Reply:
agree with `feature spec`

Reply:
updated